src/Types/sheet.ts

```diff
@@ -10,7 +10,6 @@ export interface DocumentSection {
     icon?: Displayable | ((data: UIData) => Displayable);
     action?: Displayable;
   }
-  fieldsDescription?: Displayable | ((data: UIData) => Displayable);
```

Comment:
Didn't you add this to add descriptions to the headers of fields? why remove it now?